src/containerapp/HISTORY.rst

```diff
@@ -3,6 +3,16 @@
 Release History
 ===============
 
+0.3.6
+++++++
+* Added parameter --environment to 'az containerapp list'
+* Added 'az containerapp revision label swap' to swap traffic labels
+* BREAKING CHANGE: 'az containerapp revision list' now shows only active revisions by default, added flag --all to show all revisions
+* Fixed bug with 'az containerapp up' where custom domains would be removed when updating existing containerapp
+* Fixed bug with 'az containerapp auth update' when using --unauthenticated-client-action
+* BREAKING CHANGE: 'az containerapp env certificate upload' now overwrites the existing certificate by default. Added flag --overwrite to show the prompt for confirmation of overwriting the existing certificate.
+* Fixed bug with 'az containerapp env certificate upload' where it shows a misleading message for invalid certificate name
+
 0.3.5
 ++++++
 * Add parameter --zone-redundant to 'az containerapp env create'
```

Comment:
@StrawnSC did we add examples on env create on usage?

Reply:
No we don't have any examples in the help text for --zone-redundant. I can add that now